annotation-file-utilities/tests/Makefile

```diff
@@ -32,10 +32,14 @@ DEBUG :=
 # Use this to enable some debugging.
 # DEBUG := --debug
 
-default : all
+default : diffs
```

Comment:
Why did the default change to `diffs`?  Why did you need to add the target `diffs`?

Reply:
Line 35 seems like a mistake on my part. I'll change this. 

I did add the `diffs` target to the makefile, and perhaps this should have been in a separate PR. I remember that this made it slightly easy for me to debug since I was using IntelliJ. This also seemed like the only way I would be able to run something like `make $(DIFFS)` from the command line.